README.md

```diff
@@ -54,3 +54,30 @@ For more information on anchore and anchore-cli, refer to:
 - [Anchore CLI](https://github.com/anchore/anchore-cli)
 - [Circle Orb Registry](https://circleci.com/orbs/registry/orb/anchore/anchore-engine)
 
+## About rules.json
+
+The rules.json file acts as a rules engine and enables defining rules that can accept or reject quotes. A rule is defined as an object with a title, a conditions object, and an event object. A rule specifies that if certain conditions are met, then the specified event will be generated.
+
+The rules engine used by the quoting-service is an off-the-shelf rules engine, called json-rules-engine. For detailed information on how to write rules, see the [json-rules-engine](https://github.com/CacheControl/json-rules-engine/blob/master/docs/rules.md) documentation. This page only focuses on those details that are relevant for adding support for new currencies.
+
+### Conditions
```

Comment:
Can we use a different word from conditions?

Reply:
hi @lewisdaly thanks for the comment.

Is there a reason for the change needed? This is standard terminology, used here: https://github.com/CacheControl/json-rules-engine/blob/master/docs/rules.md

Reply:
Wow that comment is from years ago... I have no recollection and don't feel strongly about this at all.